code/modules/admin/verbs/adminpm.dm

```diff
@@ -319,6 +324,10 @@
 
 	var/datum/pm_convo/convo = pms[current_title]
 	var/datum/browser/popup = new(user, window_id, "Messages", 1000, 600, src)
+
+	if(forced) // Lockout the normal close button, force the UI one
+		popup.set_window_options("can_close=0")
```

Comment:
As a note, windows have a `on-close` parameter that can be used to run a verb, but it's not really worth introducing a new verb for something as specific here.

Reply:
I was originally gonna use that, but I didnt want to add extra verbs as mentioned, and the default `onclose` verb is tied into `/atom`, not `/datum`. Could do with refctoring that at some point.